bedrock/mozorg/templates/mozorg/home/home-contentful.html

```diff
@@ -1,142 +0,0 @@
-{#
- This Source Code Form is subject to the terms of the Mozilla Public
- License, v. 2.0. If a copy of the MPL was not distributed with this
- file, You can obtain one at https://mozilla.org/MPL/2.0/.
-#}
-
-{% from "macros-protocol.html" import hero, call_out_compact, card with context %}
-
-{% from "mozorg/home/includes/macros.html" import download_banner, download_banner_secondary, fxa_banner %}
```

Comment:
Question: can all these old macros be removed?

Reply:
I'm thinking we keep the `home/includes/macros.html` file for a bit in case we may need to add them into our new homepage later on?

Reply:
Up to you 👍  The design for these macros is pretty old though, so I'd be surprised if we incorporated them back into the new design

Reply:
Oh fair, well I guess in that case we should be fine to remove. If we end up needing them, I'm sure we can fetch 'em from our git history